data/json/professions.json

```
@@ -2087,6 +2087,23 @@
       "female": [ "bra", "boy_shorts" ]
     }
   },
+  {
+    "//": "You got a random book box!",
+    "type": "profession",
+    "id": "student_with_books",
+    "name": "Library Nerd",
+    "description": "They say reading is food for brain, but those corpses seem to be coming for your brain as food. In your haste to escape, you grab the library's return box, but is there anything in there that might help you?",
```

Bit less clunky and more importantly fixes the sentence-spacing thing that's making the check fail:
```suggestion
    "description": "They say reading is food for the brain, but those corpses seem to see your brain as food.  In your haste to escape, you grabbed the library's return box, but is there anything in there that might help you?",
```